src/timeline/NowIndicator.tsx

```diff
@@ -7,14 +7,16 @@ export interface NowIndicatorProps {
   styles: {[key: string]: ViewStyle | TextStyle};
   width: number;
   left: number;
+  startOffset: number;
 }
 
 const NowIndicator = (props: NowIndicatorProps) => {
-  const {styles, width, left} = props;
+  const {styles, width, left, startOffset} = props;
 
-  const indicatorPosition = calcTimeOffset(HOUR_BLOCK_HEIGHT);
+  let indicatorPosition = calcTimeOffset(HOUR_BLOCK_HEIGHT);
 
   const nowIndicatorStyle = useMemo(() => {
+    indicatorPosition = calcTimeOffset(HOUR_BLOCK_HEIGHT) - calcTimeOffset(HOUR_BLOCK_HEIGHT, startOffset) + calcTimeOffset(HOUR_BLOCK_HEIGHT, 0, new Date().getMinutes());
```

Comment:
I think you can avoid all this calculations if you just pass the start to the calcTimeOffset function and reduce it if the start is larger then the hour:
`export function calcTimeOffset(hourBlockHeight: number, start = 0, hour?: number, minutes?: number) {
  const now = new Date();
  const h = hour ?? now.getHours();
  const m = minutes ?? now.getMinutes();
  const offset = h < start ? 0 : start;
  return ((h + m / 60) - offset) * hourBlockHeight;
}`